### PR TITLE
Add a "timed" mode to command

### PR DESCRIPTION
**Pull request by howeyc**
I made changes I talked about in Issue #5 

This will alter the temperature every 10 minutes. The temp adjusts in a linear fashion based on the time of day.

### CODE REVIEW

**Comment by howeyc** [comment]
I made adjustments based on your feedback.

Thanks.

File: sct/main.go [hunk 1]
```diff
@@ -69,7 +107,25 @@ func main() {
 			log.Println("Launched background process... pid", cmd.Process.Pid)
 			os.Exit(0)
 		} else {
-			monitorTime()
+			switch *mode {
+			case "timed":
```
Comment (d4l3k) [inline sct/main.go:125]: Thanks for making the refactors. However, this still has the issue that it'll fork to the background and silently die if there are bad values. You should move the time parsing code to before the if/else statement (L58).

**Comment by howeyc** [comment]
ping!

Not sure if you got a notification when I made latest commit. No rush, just checking.

**Comment by d4l3k** [comment]
Yeah, sorry about that. Classes are finishing up for me and I got a bit distracted.

Looks good to me!

**Pull request merged**